lib/screens/v3_api/cache.ex

```diff
@@ -0,0 +1,60 @@
+defmodule Screens.V3Api.Cache do
```

Comment:
This whole module is really understandable with the structure of realtime vs. static and the comments to explain reasoning!!